cerbero/utils/shell.py

```diff
@@ -782,3 +782,12 @@ def run_until_complete(tasks, max_concurrent=16):
     else:
         result = loop.run_until_complete(tasks)
     return result
+
+
+def remove_empty_dirs(path):
+    '''
+    Remove all empty dirs within path
+    @param path: the path to remove empty dirs from
+    @type path: str
+    '''
+    new_call('find {} -type d -delete'.format(path))
```

Comment:
If there are some non-empty dirs, it will output errors and `find` tool will return 1 as error code.
Maybe it is not an issue, I just wanted to state it out just in case ;)

Reply:
Good catch! I'll add the `-empty` argument to sort it out